core/src/consensus/light_client/verification.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use ccrypto::blake256;
+use ctypes::CompactValidatorSet;
+use merkle_trie::proof::{verify, CryptoProof, CryptoProofUnit};
+
+use super::LightHeader;
+use super::Seal;
+
+pub fn verify_header(
+    previous_header: &LightHeader,
+    vset: &CompactValidatorSet,
+    new_header: &LightHeader,
+    seal: &Seal,
+) -> bool {
+    assert_eq!(vset.0.len(), seal.0.len());
```

Comment:
Could you give names to the .0 and .1?
Like: 

```
let validators = vset.0;
let signatures = seal.0;
```

Reply:
You can use destructuring assignment.
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=249d0ab587196c08d35e5ac6124deff1